<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Countdown Timer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_reset">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>270</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minutes_plus">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minutes_minus">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mem_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>240</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>30</y>
      <width>570</width>
      <height>190</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>170</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_seconds_plus">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>240</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_settings">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>320</y>
      <width>210</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_seconds_minus">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mem_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>240</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mem_1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>270</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mem_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>320</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>100</y>
      <width>135</width>
      <height>116</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="title">
     <string/>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="actionDonate"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDonate">
   <property name="text">
    <string>Donate</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
